tests/test_repodict.py

```diff
@@ -16,18 +18,20 @@
 #
 
 from __future__ import unicode_literals
-from tests import support
-from tests.support import TestCase
+
 import dnf.repodict
 
+import tests.support
+from tests.support import TestCase
+
 
 class TestRepoDict(TestCase):
```

Comment:
this should be class (tests.support.TestCase): for consistency with all other files